README.md

```diff
@@ -1 +1,185 @@
-# BLOG-app.
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# Blog App <a name="about-project"></a>
+
+A **Ruby blog app** The web application is designed to enable users to create, edit, and publish blog posts. It is developed using the Ruby on Rails framework, a popular choice known for its speed and user-friendly features in web application development.
+
+Additionally, the application functions as a standard blog website, offering a comprehensive and functional platform. Users can browse a collection of articles and interact with them by leaving comments and expressing their appreciation through features like liking posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Ruby on Rails</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Rails</a></li>
+    <li><a href="https://rubocop.org/">Rubocop</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features of this project includes.
+
+- **Display Users**
+- **Display Posts**
+- **Display Likes on Posts**
+- **Display Comments on Posts**
```

Comment:
- [ ] Kindly make sure that your **Key feature section of the README file describe the current project milestone .**